client/src/selectors/schedules.js

```diff
@@ -1,5 +1,32 @@
 import moment from 'moment';
 
+import Colors from '../themes/Colors';
+
+export const isSelectorDisabled = (selectionSegments, status, startTime, endTime) => {
```

Comment:
it'd be so great to get some tests on this

Reply:
Agreed - that's why I put the logic in this area